docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -576,6 +584,59 @@ the two columns or sets of columns must have the same characteristics (data type
 the foreign key in _table_ is the column being defined; with the foreign key clause, the foreign key is the column or set of columns specified in
 the foreign key clause. for information about _ref-spec_, see references _ref-spec_ not enforced.
 
+<<<
+* ENABLE ALL [UNIQUE] INDEXES
+
+** ENABLE ALL UNIQUE INDEXES
++
+enables all unique indexes on the table.
+
+** ENABLE ALL INDEXES
++
+enables all indexes (including normal indexes and unique indexes) on the table.
+
+* _index-name_
+** _normal-index-name_
++
+is equal to _index-name_, which is an SQL identifier that specifies the simple name for the index. You cannot qualify index-name with its schema name. 
+Indexes have their own name space within a schema, 
+so an index name might be the same as a table or constraint name. However, no two indexes in a schema can have the same name.
+
+** _unique-index-name_
++
+is the name of the unique index.
+
+* ENABLE INDEX _normal-index-name_ 

```

Comment:
Again, I would not distinguish between unique and non-unique indexes here. Simply say "ENABLE INDEX _index-name_ <blank lines> enables an index on the table" is sufficient.

Reply:
Thanks @DaveBirdsall, I've udpated :)